app/views/apply/students.html.erb

```diff
@@ -20,7 +20,7 @@
       </p>
     </div>
     <div class="small-12 medium-3 columns" id="apply-button">
-      <div class="button">Apply Now</div>
```

Comment:
Took these out because I think we should "force" them to "learn more" before applying